buildkite/scripts/build-client-sdk.sh

```diff
@@ -2,5 +2,20 @@
 
 set -o pipefail
 
+echo "--- Make client sdk"
 mkdir -p /tmp/artifacts
 make client_sdk 2>&1 | tee /tmp/artifacts/buildclientsdk.log
+
+echo "--- Get yarn"
+sudo apt install apt-transport-https
+curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash -
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
```

Comment:
these steps seem unreasonable to repeat on each run, can yarn be included in whatever docker image these jobs run from?

Reply:
Yes, I suppose you're right. I shall pay the docker rebuild penalty